doc/quickstart.rst

```diff
@@ -383,3 +383,29 @@ The directory in which you invoked *SMAC* now contains a new folder called **SMA
 The .json file contains the information about the target algorithms *SMAC* just executed. In this file you can see the *status* of the algorithm run, *misc*, the *instance* on which the algorithm was evaluated, which *seed* was used, how much *time* the algorithm needed and with which *configuration* the algorithm was run.
 In the folder *SMAC* generates a file for the runhistory, and two files for the trajectory.
 
+
+.. _hydra-example:
+
+Hydra on Spear-QCP
+------------------
+
+For this example we use *Hydra* to build a portfolio on the same example data presented in `Spear-QCP`__.
+Hydra is a portfolio builder that aims to build a portfolio by iteratively adding complementary configurations to the
+already existing portfolio. In the first iteration Hydra runs standard *SMAC* to determine a well performing configuration
+across all instances as a starting point for the portfolio. In following iterations Hydra only considers configurations
+that can improve the portfolio performance. Configurations that perform poor on instances which are already solved well
```

Comment:
Sorry, but the sentence is still not acceptable. Maybe we should replace the entire sentence?
I would recommend that we don't write what Hydra is *not* doing, but what it actually does. 
E.g., something along the following lines: "Hydra evaluates configurations in terms of performing better than the current portfolio."